extensions.json

```
@@ -809,6 +809,12 @@
       "id": "lintangwisesa.arduino",
       "repository": "https://github.com/LintangWisesa/VScode-Arduino-Theme"
     },
+    {
+      "id": "liqube.theme-liqube-dark",
+      "repository": "https://github.com/liqube/liqube-dark-vscode",
+      "version": "1.0.5",
+      "checkout": "master"
```

Unfortunately, this repository doesn't provide any GitHub Releases or release tags. So, the best remaining option is to publish straight from the default branch (no need to explicitly check it out, and no need to pin a specific version):

```suggestion
      "repository": "https://github.com/liqube/liqube-dark-vscode"
```